specificatie/openapi.yaml

```diff
@@ -573,7 +573,7 @@ components:
         example: '0226200000038923'
 
     adresseerbaarobjectidentificatie:
-      description: "De identificatie van een adresserbaar object uit de BAG. Dit kan een verblijfsobject, een standplaats of een ligplaats zijn."
+      description: "De identificatie van een adresserbaar object uit de BAG. Deze is 16 cijfers lang en kan een verblijfsobject, een standplaats of een ligplaats zijn."
```

Comment:
Een consumer kan toch aan de pattern zien dat deze parameter 16 lang is ? En waarom voeg je dit dan alleen bij deze parameter toe in de description terwijl er nog 6 parameters zijn die 16 lang zijn waarbij deze opmerking niet toegevoegd wordt ?

Reply:
Ik heb dit juist toegevoegd omdat dat overal al bij stond, alleen hier niet. Dus om e.e.a. consistent te maken met zo min mogelijk effort heb ik het alleen hier veranderd.